client/app/vps/additional-disk/vps-additional-disk.html

```diff
@@ -9,7 +9,7 @@
     </div>
 
     <div data-ng-if="$ctrl.hasAdditionalDiskOption && !$ctrl.loaders.disk">
-        <p data-ng-if="$ctrl.canOrder" data-translate="vps_tab_additional_disk_count"></p>
+        <p data-ng-if="$ctrl.canOrder()" data-translate="vps_tab_additional_disk_count"></p>
```

Comment:
you could remove canOrder function from controller and replace by this.additionnalDisks.length > 0